_analyzers/standard.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: default
+title: Standard analyzer
+nav_order: 40
+---
+
+# Standard analyzer
+
+The `standard` analyzer is the default analyzer that is used when no other analyzer is specified. It is designed to provide a basic and efficient approach for generic text processing.
+
+This analyzer consists of the following tokenizers and token filters:
+
```

Comment:
Line 13: I believe we had been using "split text at" (as opposed to "on"), but "on" started appearing in recent docs. Please ensure global consistency.

Reply:
Will address this as a separate PR globally.